src/app/pages/details/instructor-resource-tab/partners/partners.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import routerBus from '~/helpers/router-bus';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import './partners.css';
+
+function Blurb({blurb, badgeImage, onClick}) {
+    return (
+        <a className="blurb" href={blurb.url} onClick={onClick}>
+            <div className="logo">
+                <img src={blurb.image} alt="" />
+            </div>
+            {
+                blurb.verifiedFeatures &&
+                    <div className="badge">
+                        <img className="background" src={badgeImage} alt="verified" />
+                        <FontAwesomeIcon className="checkmark" icon="check" />
+                        <div className="tooltip right">
+                            {blurb.verifiedFeatures}
+                        </div>
+                    </div>
+            }
+            <div className="name">{blurb.name}</div>
+            <div className="tags">
+                {
+                    blurb.cost &&
+                        <div className="info">
+                            <span className="label">Cost:</span>
+                            {blurb.cost}
+                        </div>
+                }
+                {
+                    blurb.type &&
+                        <div className="info">
+                            <span className="label">Type:</span>{blurb.type}
+                        </div>
+                }
+            </div>
```

Comment:
Like I noted above, I'd probably make the a `VerifiedFeatures`, `Cost` and `Type` component and render them here.  Each component would return null if the `blurb` didn't have that property